src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -2542,10 +2538,10 @@ void ColorControlServer::startUpColorTempCommand(EndpointId endpoint)
                 if (status == EMBER_ZCL_STATUS_SUCCESS)
                 {
                     // Set ColorMode attributes to reflect ColorTemperature.
-                    uint8_t updateColorMode = EMBER_ZCL_COLOR_MODE_COLOR_TEMPERATURE;
+                    uint8_t updateColorMode = to_underlying(ColorControl::ColorMode::kColorTemperature);
                     Attributes::ColorMode::Set(endpoint, updateColorMode);
```

Comment:
ColorMode::Set should be taking the enum type, I would think... If it's not, is there a followup filed to fix that?  Might need to wait on https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/6174

Reply:
These were sed updates (+ manual fixes for to_underlying at times) ... I did not update logic. I think actually using some types would be better, but then backwards compatibility may be an issue if we start forcing those.

Reply:
Updated everal other types, but not `Set` because spec does not yet name that type and that would be API incompatible (and I am not sure we have a story for that yet). I agree they should be taking an enum, however unfortunately this is what we have now.